api/src/services/listing.service.ts

```diff
@@ -1210,20 +1210,28 @@ export class ListingService implements OnModuleInit {
 
     const newListingData: ListingCreate = {
       ...mappedListing,
-      name: dto.name,
+      applicationMethods: applicationMethods,
       assets: [],
-      status: ListingsStatusEnum.pending,
+      listingsBuildingSelectionCriteriaFile:
+        mappedListing.listingsBuildingSelectionCriteriaFile
```

Comment:
**nitpick (non-blocking):** Could you move this logic the newListingData to match the pattern of handling other assets?

Reply:
Would want to make this change in core since tis is a release PR